### ISSUE TITLE
po2ts fails with "str not buffer" error on py3

### ISSUE THREAD

**Issue by Artoria2e5** [translate/translate#3602]
Test file: see #3601 

```
$ po2ts octave.po oct.ts
processing 1 files...
po2ts: WARNING: Error processing: input octave.po, output oct.ts, template None: 'str' does not support the buffer interface
[###########################################] 100%

$ python3 --version
Python 3.4.5
```

**Comment by Artoria2e5** [comment]
Error since 2.0.0b7 (or earlier), still happens on 2.1.0

**Comment by unho** [comment]
Fixed by #3603, so closing.
